### DISCUSSION TITLE
Questions for Serious security issue

### DISCUSSION THREAD

**Discussion by KurstenDavis** [community/community#108595] [Enterprise]
Can anyone advise me how best to report stop and make public aware of cyber security issue posing to be legitimate using GitHub constant content through an Enterprise device to do it? It is of extreme serious nature and if anyone needs pictures or documents that I have found across all devices hidden in not only Google Drive but Microsoft everything and again this account in which I am making this note on, I never even created nor new it existed but yet was created in 2020 and even has its own docks and Repository. I haven't the first clue about any of this or what I'm looking at. All I know is he has gone way past developer settings and is an engineer mode using stack overflow, ?!?res-overlays??!? ..written in java script with all "Read Me" docs & dource coding saved as xml files (impossible to read or convert) and an ??!?ADB..ABD??!?shell which trips my bootloader anytime my phone restarts and loads his manifest instead.... called mtk6888 or something.  But I have years worth of these documents and live video and audio in which he would record and set on timers every 6 hours to record a Solid 5 minutes anything and everything around me video and audio. I know the absolute realm of how illegal this is but there's got to be a way to let Google Red Hat Society constant content GitHub Android developer and most of all Enterprise aware that they're open source licensing projects are just as harmful as they are meant for free knowledge and sharing. Every one of those websites especially this one give him a perfect template that he just downloads and paste as a user certificate and violates every personal aspect of my being by third party open source licensing by Apache. I know that's quite lengthy and probably all over the place for solid tech savs as yourselves that know how F'd up of what is going on, all I know is it's been going on for 6 years and a lot of ramifications and consequences due to it and caused from it has done extreme irreversible damage mostly estranged communication for over two and a half years with my daughters and it strained meaning non-existent..

**Comment by Timmmy-nlb** [comment]
Very hard to read without some structure...
Is this a GitHub problem or a Microsoft / Google?

**Reply by Lanrad23** [reply]
Well, in cases like this I can only suggest to keep detailed records of all evidence, including screenshots, documents, code snippets, and any other relevant information. Organize these documents carefully for easy reference.

**Comment by BlueSky-2024** [comment]
Addressing serious security issues requires immediate action and the right expertise. For insights on protecting your data, check out https://www.wizlynxgroup.com/de/.

**Comment by BlueSky-2024** [comment]
It’s essential to address security concerns seriously, especially when dealing with tools that collect personal data. For those interested in a practical application that handles such data with care, the Snow Day Calculator might be worth a look. It uses updated weather information and localized data to provide predictions, and its development team focuses on maintaining strong security measures. You can find more about it here: https://snowdaycalculatorpro.com/.
